docs/documentation/features/frameworkUtilities.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: documentation
+---
+
+# Provided Services
+
+In this chapter useful services of the Eclipse SmartHome project are described. 
+
+## Network Address Service
+
+A user can configure his default network address via Paper UI under `Configuration -> System -> Network Settings`.
+To obtain this configured address the `ThingHandlerFactory` needs a `service reference` to the `NetworkAddressService` in its `MyHandlerFactory.java`:
+
+```java
+@Reference
```

Comment:
Do you really think we need to explain how OSGi DS works for every single service we offer?
Wouldn't it be more important to state that the NetworkAddressService IS an OSGi service that can be used?